src/components/shared/SearchCard/SearchCardAction/SearchCardAction.js

```diff
@@ -1,7 +1,7 @@
-import { memo } from 'react'
+import { memo, useEffect } from 'react'
```

Comment:
Without `useEffect` and calling `pollDatasetAccession` will results in run-time error (`Too many re-renders`). 

Please see Vercel preview [here](https://refinebio-gr8s0h2ob-ccdl.vercel.app/search) (at 5804932e336bef1d6875eca092d248b44d26a1cc).

Reply:
You can fix that by just adding a guard on `pollDatasetId`. Just return early if the passed in `datasetId` is already `polledDatasetId` since you can presume you already have a timer for that in place.

Reply:
See update at 89f9daea7d91711677556287cf635924d38a667d.